src/Catty/ViewController/CatrobatTableViewController/CatrobatTableViewController.m

```diff
@@ -407,4 +407,40 @@ - (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIVi
     [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
 }
 
+#pragma mark - open project from file
+-(void)addProjectFromFile:(NSURL*)url
+{
+    [Util askUserForUniqueNameAndPerformAction:@selector(addProjectFromFileWithName:withUrl:)
```

Comment:
Like in Catroid the import should be done using the name in the project's header directly. If the name exists already, the name should be "Name (1)" and so on (this functionality already exists). Please do the actual import job in the `ProjectManager` instead of the view controller (easier to test).